how-to/software/upgrade-your-release.md

```diff
@@ -1,41 +1,50 @@
 
-## Upgrade paths
+## Understanding Upgrade Paths
+You can only upgrade from one LTS release directly to the **next sequential LTS release**. For example, if you are on Ubuntu 16.04 LTS, you can upgrade to Ubuntu 18.04 LTS.
+However, you cannot skip releases (e.g. jump from 16.04 LTS to 20.04 LTS).If you need to reach a later LTS, you will have to upgrade in stages: first to Ubuntu 18.04 LTS, then to Ubuntu 20.04 LTS, and so on.
 
-Ubuntu supports the ability to upgrade from one LTS to the next in sequential order. For example, a user on Ubuntu 16.04 LTS can upgrade to Ubuntu 18.04 LTS, but cannot jump directly to Ubuntu 20.04 LTS. To do this, the user would need to upgrade twice: once to Ubuntu 18.04 LTS, and then upgrade again to Ubuntu 20.04 LTS.
 
 ## Pre-upgrade checklist
 
-To ensure a successful upgrade, review the following items:
+Before starting a mjor release upgrade, it's important to prepare our system to ensure a smooth transition. This steps are essential, so we need to review the following items:
 
-* Check the release notes (for the new release) for any known issues or important changes. Release notes for each release are found on the [Ubuntu Wiki releases page](https://wiki.ubuntu.com/Releases).
+* **Review Releases notes:** Always check the **release notes** for the new Ubuntu version we are moving to. This can be found on the [Ubuntu Wiki Releases Page](https://wiki.ubuntu.com/Releases).
 
-* Fully update the system. The upgrade process works best when the current system has all the latest updates installed. You should confirm that these commands complete successfully and that no further updates are available. We also suggest rebooting the system after all the updates are applied, to ensure the latest kernel is being run. To upgrade, run the following commands:
+* **Fully update the current system:** The release upgrade process works best when the current system has all the latest updates installed. This is a standard **package upgrade**, **apt update** will refresh package lists (sync our package index with the upstream sources), and **apt upgrade** will install the latest versions of installed packages.
 
-  ```bash
-  sudo apt update
-  sudo apt upgrade
-  ```
+  1. Run these commands to ensure everything is up to date:
 
-* Check that there is enough free disk space for the upgrade. Upgrading a system will include downloading new packages, which is likely to be on the order of hundreds of new packages. Systems with additional software installed may therefore require a few gigabytes of free disk space.
+      ```bash
+      sudo apt update
+      sudo apt upgrade
+      ```
+  2. Confirm both commands complete successfully and no further updates are available.
 
-* The upgrade process takes time to complete. You should have dedicated time to participate in the upgrade process.
+  3. After applying all updates, **reboot your system** to ensure the latest kernel is running.
```

Comment:
Replace:

   3. After applying all updates, **reboot your system** to ensure the latest kernel is running.
 
With:

 3. After applying all updates, it may be necessary to **reboot your system**. The release upgrade process will let you know if that's needed, but you can also check manually before: if the file `/run/reboot-required` exists, then you will need to reboot.